src/TestRotations.java

```diff
@@ -0,0 +1,55 @@
+//: object/TestRotations.java
+
+import junit.framework.*;
+
+/**
+ * JUnit Class to test the rotation methods from the Quaternion and RotationMatrix classes.
+ *
```

Comment:
I don't get it, you mean rotating first with a quaternion and then "unrotate" with a matrix and see if I return to the original vector?